src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+import java.util.*;
+
```

Comment:
I really like how you have managed to have the appropriate standard spacing (e.g. before a '{') in your code. Continue the good effort!